### PR TITLE
Data management

### PR DESCRIPTION
**Pull request by NicolasFujisawa**
@jasondavindev

### CODE REVIEW

File: data_quality_governance/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- Software used, include the version of the software, if future users are using a different version, they can work through the differences and software issues that might occur, its also a good practice keep the software, version and the options to be installed always an standard.
+
+- Context, it is essential to give any context to the project, why it was created, if hypotheses were trying to be proved or disproved, etc.
+
+---
+
+## Data quality trust in security and privacy
+
+We have a culture who priorize the security of our data with strong privacy standards to build a relationship with clients.
```
Comment (jasondavindev) [inline data_quality_governance/README.md]: Talk about data access management, introduction to LGPD (pt br)

File: data_quality_governance/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Data Quality Governance
+
+Data quality governance is a discipline that aims to manage and watch over our company data, so that data can be transformed into business value and support strategic decisions. For this, data quality governance uses methodologies, processes, professionals and tools.
```
Comment (jasondavindev) [inline data_quality_governance/README.md:3]: Impersonal sentences

File: data_quality_governance/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+- Prefix should make sense and be easy to understand.
+
+### Date and Time conventions
+
+To save dates, the standard format is `YYYY-MM-DD`.
+
+And to save time, the format is `HH:MM:SS:ZZZ`.
+
+---
```
Comment (toledompm) [inline data_quality_governance/README.md]: :bicyclist: I think it's worth mentioning ISO 8601 here

https://www.iso.org/iso-8601-date-and-time-format.html

**Review by toledompm — APPROVED** [review]

**Comment by jasondavindev** [comment]
Pode fechar

**Pull request closed without merging**